src/attr/_funcs.py

```diff
@@ -13,6 +13,7 @@ def asdict(
     filter=None,
     dict_factory=dict,
     retain_collection_types=False,
+    hook=None,
```

Comment:
Same here, I'm afraid we need long arg names. :)

Reply:
`value_serializer()`?

Reply:
Yeah, better I think.